static/app/views/issueDetails/actions/index.spec.tsx

```diff
@@ -168,126 +169,123 @@ describe('GroupActions', function () {
-      userEvent.click(await screen.findByTestId('delete-issue'));
-
-      const modal = screen.getByRole('dialog');
-      expect(
-        within(modal).getByText(/Deleting this issue is permanent/)
-      ).toBeInTheDocument();
-
-      userEvent.click(within(modal).getByRole('button', {name: 'Delete'}));
-
-      expect(deleteMock).toHaveBeenCalled();
-      expect(browserHistory.push).toHaveBeenCalledWith({
-        pathname: `/organizations/${org.slug}/issues/`,
-        query: {project: project.id},
-      });
+    const updateMock = MockApiClient.addMockResponse({
+      url: `/projects/${org.slug}/${project.slug}/issues/`,
+      method: 'PUT',
+      body: {},
     });
-
-    it('resolves and unresolves issue', () => {
-      const issuesApi = MockApiClient.addMockResponse({
-        url: `/projects/${org.slug}/project/issues/`,
-        method: 'PUT',
-        body: {...group, status: 'resolved'},
-      });
-
-      const {rerender} = render(
+    render(
+      <Fragment>
+        <GlobalModal />
         <GroupActions
           group={group}
           project={project}
           organization={org}
           disabled={false}
-        />,
-        {organization: org}
-      );
+        />
+      </Fragment>,
+      {organization: org}
+    );
 
-      userEvent.click(screen.getByRole('button', {name: 'Resolve'}));
+    userEvent.click(screen.getByLabelText('More Actions'));
+    userEvent.click(await screen.findByText('Share'));
 
-      expect(issuesApi).toHaveBeenCalledWith(
-        `/projects/${org.slug}/project/issues/`,
-        expect.objectContaining({data: {status: 'resolved', statusDetails: {}}})
-      );
+    const modal = screen.getByRole('dialog');
+    expect(within(modal).getByText('Share Issue')).toBeInTheDocument();
+    expect(updateMock).toHaveBeenCalled();
+  });
 
-      rerender(
+  it('opens delete confirm modal from more actions dropdown', async () => {
+    const org = {
+      ...organization,
+      access: [...organization.access, 'event:admin'],
+    };
+    MockApiClient.addMockResponse({
+      url: `/projects/${org.slug}/${project.slug}/issues/`,
+      method: 'PUT',
+      body: {},
+    });
+    const deleteMock = MockApiClient.addMockResponse({
+      url: `/projects/${org.slug}/${project.slug}/issues/`,
+      method: 'DELETE',
+      body: {},
+    });
+    render(
+      <Fragment>
+        <GlobalModal />
         <GroupActions
-          group={{...group, status: 'resolved'}}
+          group={group}
           project={project}
           organization={org}
           disabled={false}
         />
-      );
+      </Fragment>,
+      {organization: org}
+    );
 
-      const resolvedButton = screen.getByRole('button', {name: 'Resolved'});
-      expect(resolvedButton).toBeInTheDocument();
-      userEvent.click(resolvedButton);
+    userEvent.click(screen.getByLabelText('More Actions'));
+    userEvent.click(await screen.findByTestId('delete-issue'));
```

Comment:
can we check for the button label https://github.com/getsentry/sentry/blob/8e00dcdf463d916b9ca79ddbe13e99f161d58db1/static/app/views/issueDetails/actions/index.tsx#L381?

```suggestion
    userEvent.click(await screen.findBylabelText('Delete'));
```